src/main/resources/CU-ApplicationResources.properties

```diff
@@ -354,9 +354,9 @@ error.document.paymentmethod.flagrequired=At least one of the {0}, {1}, and {2}
 error.document.paymentmethod.not.pdp.and.interdept=A payment method may not be both interdepartmental and processed by PDP.
 error.document.paymentmethod.no.bank.when.interdept=A bank code may not be entered when the payment method is an interdepartmental charge.
 
-# Additional DV document messages - KITT-592
-error.document.disbursementvoucher.bank.required=A bank code is required for payment method {0}.
-warning.document.disbursementvoucher.bank.not.required=Bank codes are not required on payment method {0}, the entered value has been removed.
+# Additional bank-related document messages - KITT-592
```

Comment:
Can this comment just be removed, I don't think it adds much value any longer

Reply:
Ok, I'll remove the comment.